docs/config.toml

```diff
@@ -27,7 +27,7 @@ pygmentsUseClasses = true
   IsStable = true
 
   # Flag to indicate whether an outdated warning should be shown.
-  ShowOutDatedWarning = false
+  ShowOutDatedWarning = true
```

Comment:
I'm not sure if we should mark 1.16 as an outdated version. My understanding is that only end-of-life versions like 1.15 and 1.16 are "outdated" 🤔

Reply:
That's actually a good point. The warning itself states
> This documentation is for an out-of-date version of Apache Flink. We recommend you use the latest [stable version](https://nightlies.apache.org/flink/flink-docs-stable/).

which backs your claim. And it also makes sense because we can still do updates to the docs of 1.16.

The release documentation says "Set show_outdated_warning: true in docs/config.toml in the branch of the previous Flink version" which is a bit unclear. I'm going to close this branch and update the release docs to make this more precise.